### DISCUSSION TITLE
Finding a technical solution is not easy

### DISCUSSION THREAD

**Discussion by DocSmirc** [singlesourcepub/community#56] [Q&A]
Hi guys,

I am new to the Single Source Publishing bubble and am looking for a solution.

I have been publishing content on my website (wordpress) for years and update my content regularly. After some research I stumbled across the concept of single source publishing and I think it's just the thing for me. 

My wish would be the approach to feed from one formatted content (tables, page breaks and graphics) but as one source of truth , then with "one click" different formats:

PDF: with the previously specified format

HTML: plain text, tables

KDP: with the previously specified format

Because I regularly update my content, intermediate steps like Word to Indesign are not an alternative. My project is currently ~1500 standard pages in size, which I don't want to reformat with every correction of a typo.

Markdown is not a problem for me and I like it. I have already done projects in "homebrewery" and "GMBinder".
Does anyone have a simple solution for me? Many thanks in advance.

Sebastian

**Comment by castedo** [comment]
> then with "one click" different formats

Does "with one click" mean you seek point-and-click user-friendly tools and not Linux command-line oriented type-and-hit-enter tools?

> Does anyone have a simple solution for me?

From your summary description this does not sound like something that has a simple solution. Sorry that I don't have an optimistic answer.

**Reply by DocSmirc** [reply]
A one click solution would be the wish, but I don't think there is such a thing. 

Scripting/comando line is not really my thing. sorry. 

But if the elements can be connected via script, it would be my solution, wouldn't it? The question is, how much manual effort does each publishing cycle mean?

**Reply by castedo** [reply]
Connecting the elements via script can result in essentially zero manual effort for each publishing cycle. For example, the elements documented on manubot.org illustrate automated publishing cycles with zero manual effort.

However I suspect you will find the connecting of these elements via scripts not at all simple if scripting/commando line is not your thing.

**Reply by DocSmirc** [reply]
mh....possible.
please let me check. 
Thank you for the answer.

**Comment by kaveh1000** [comment]
Well, for the scholarly publishing industry, i.e. scientific papers, the
big publishers have demanded "XML first" for some 20 years, so XML is
created (usually using the JATS DTD), then with a single click all other
formats, including PDF are created. My company does that as a service, but
I am afraid we do not have an open source solution for you. :-(

On Sun, 30 Apr 2023 at 14:01, Castedo Ellerman ***@***.***>
wrote:

> then with "one click" different formats
>
> Does "with one click" mean you seek point-and-click user-friendly tools
> and not Linux command-line oriented type-and-hit-enter tools?
>
> Does anyone have a simple solution for me?
>
> From your summary description this does not sound like something that has
> a simple solution. Sorry that I don't have an optimistic answer.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/singlesourcepub/community/discussions/56#discussioncomment-5766063>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBPDK52WF3VFMYT77HF343XDZPDFANCNFSM6AAAAAAXQ3X2Y4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> .com>
>


-- 
Kaveh Bazargan PhD
Director
River Valley Technologies <http://rivervalley.io> ● Twitter
<https://twitter.com/rivervalley1000> ● LinkedIn
<https://www.linkedin.com/in/bazargankaveh/> ● ORCID
<https://orcid.org/0000-0002-1414-9098> ● @***@***.***
***@***.***>
*Accelerating the Communication of Research*

*
<https://www.linkedin.com/posts/bazargankaveh_ismte-innovation-award-recipient-kaveh-bazargan-activity-7039348552526921728-XAEB/?utm_source=share&utm_medium=member_desktop>
 [image:
https://rivervalley.io/gigabyte-wins-the-alpsp-scholarly-publishing-innovation-award-using-river-valleys-publishing-technology/]
<https://rivervalley.io/gigabyte-wins-the-alpsp-scholarly-publishing-innovation-award-using-river-valleys-publishing-technology/>*

**Reply by DocSmirc** [reply]
thank you for the answer. ;-)
